components/support/error/Cargo.toml

```diff
@@ -6,7 +6,6 @@ edition = "2018"
 license = "MPL-2.0"
 
 [dependencies]
-thiserror = "1.0"
```

Comment:
This is okay to remove because the macro assumes the **consumer** crate has `thiserror`, it never used its own `thiserror` dependency